.github/workflows/_bazel-build-test.yml

```diff
@@ -164,13 +164,19 @@ jobs:
           )
           docker exec -t "${container_name}" sh -c 'sudo chown -R jenkins . && sudo chown -R jenkins /dev && .jenkins/pytorch/test.sh && cp -Lr ./bazel-testlogs ./test/test-reports'
 
+      - name: Print undeleted logs
```

Comment:
Also I wonder if this looks nicer if it's run as part of `teardown` routine, i.e. `pytorch/test-infra/.github/actions/teardown-linux@main`.  But then this is pretty much PyTorch-only step, so adding it there is optional

Reply:
I wanted to put it as close to the test step as possible b/c I got confused when it was placed in other places, but it is kind of annoying to have to copy this step everywhere so I'm not sure what to do about this

Reply:
I'm good with leaving this as it is at the moment.  May be when there are more generic steps that we can group together to run after testing finishes, i.e. uploading artifacts, cleaning up, printing logs, etc, we can create a new reusable action to include them all.  It could be a good BE task from project Nova point of view (to be reusable across all PyTorch repos)